hera_cal/abscal.py

```diff
@@ -298,9 +303,14 @@ def TT_phs_logcal(model, data, antpos, wgts=None, refant=None, verbose=True, zer
         is identically zero across all frequencies. If None, use the first key in data as refant.
 
     antpos : antenna position vectors, type=dictionary
-          keys are antenna integers, values are 2D
-          antenna vectors in meters (preferably centered at center of array),
-          with [0] index containing east-west separation and [1] index north-south separation
+          keys are antenna integers, values are antenna positions vectors
+          (preferably centered at center of array). If assume_2D is True, it is assumed that the 
+          [0] index contains the east-west separation and [1] index the north-south separation
+          
+    assume_2D : type=boolean, [default=False]
+                If this is true, all dimensions of antpos beyond the first two will be ignored.
```

Comment:
indentation

Reply:
I don't understand... this docstring doesn't have an obvious pattern for indentation

Reply:
you've got three indents here, all the others kwarg descriptions are two indents

Reply:
They are actually all over the place. model is 8 spaces, data is 7 spaces, wgts is 7, refant is 4, antpos is 6, assume_2D is 12, etc.

Reply:
I'll make them all line up with the first word after the :, which I think is the closest thing we have to a standard.